translatorbot/daily-llm.py

```diff
@@ -0,0 +1,219 @@
+import argparse
+import os
+import random
+import re
+import time
+from threading import Thread
+
+from daily import EventHandler, CallClient, Daily
+from datetime import datetime
+from dotenv import load_dotenv
+
+import config
+
+from orchestrator import Orchestrator
+from scenes.story_intro_scene import StoryIntroScene
+from scenes.start_listening_scene import StartListeningScene
+from auth import get_meeting_token, get_room_name
+
+load_dotenv()
+
+class DailyLLM(EventHandler):
+    def __init__(
+            self,
+            room_url=os.getenv("DAILY_URL"),
+            token=os.getenv('DAILY_TOKEN'),
+            bot_name="Translator",
+            language="french", # keep the generated images to a certain theme, like "golden age of illustration," "talented kid's crayon drawing", etc.
```

Comment:
The comment here is a little confusing since the property is called language, but the comment references images? Can we clarify a little?